product_brand/__manifest__.py

```diff
@@ -8,18 +8,20 @@
 
 {
     'name': 'Product Brand Manager',
-    'version': '9.0.1.0.0',
+    'version': '10.0.1.0.0',
     'category': 'Product',
     'summary': "Product Brand Manager",
     'author': 'NetAndCo, Akretion, Prisnet Telecommunications SA'
               ', MONK Software, Odoo Community Association (OCA)'
               ', SerpentCS Pvt. Ltd.',
     'license': 'AGPL-3',
-    'depends': ['product'],
+    'depends': [
+        'sale',
```

Comment:
I just depend on sale because of the menu item to keep it in the same place as older versions, if somebody have a better place, I will be glad to position the menu there and change that dependency